py/server/deephaven/table.py

```diff
@@ -55,8 +55,11 @@
 def _j_py_script_session() -> _JPythonScriptSession:
     j_execution_context = _JExecutionContext.getContext()
     j_query_scope = j_execution_context.getQueryScope()
-    j_script_session_query_scope = strict_cast(j_query_scope, _JScriptSessionQueryScope)
-    return strict_cast(j_script_session_query_scope.scriptSession(), _JPythonScriptSession)
+    try:
+        j_script_session_query_scope = strict_cast(j_query_scope, _JScriptSessionQueryScope)
+        return strict_cast(j_script_session_query_scope.scriptSession(), _JPythonScriptSession)
+    except Exception as e:
+        return None
```

Comment:
I don't totally understand the exception case here:
1) Do we need to log anything?
2) Is `Exception` so wide that we might swallow problem cases?